kornia/augmentation/_2d/geometric/crop.py

```diff
@@ -171,7 +171,10 @@ def apply_transform_box(
     def apply_transform(
         self, input: Tensor, params: Dict[str, Tensor], flags: Dict[str, Any], transform: Optional[Tensor] = None
     ) -> Tensor:
-        padding_size = params.get("padding_size").unique(dim=0).cpu().squeeze().numpy().tolist()
+        _padding_size = params.get("padding_size")
+        padding_size: Optional[List[int]] = None
+        if _padding_size is not None:
+            padding_size = _padding_size.unique(dim=0).cpu().squeeze().numpy().tolist()
```

Comment:
This may be more readable/clear
```suggestion
        padding_size: Optional[List[int]] = None
        if "padding_size" in params and isinstance(params["padding_size"], Tensor):
            padding_size = params["padding_size"].unique(dim=0).cpu().squeeze().numpy().tolist()
```